doc/lua_api.txt

```diff
@@ -4094,6 +4120,24 @@ The Biome API is still in an experimental phase and subject to change.
     --  ^ Uses texture (string)
         playername = "singleplayer"
     --  ^ optional, if specified spawns particle only on the player's client
+	material_type_param = minetest.pack_texture_blend_func(
+			minetest.ebf.src_alpha, minetest.ebf.one,minetest.emfn.modulate_1x, 
+			minetest.eas.texture + minetest.eas.vertex_color),
+    --  ^ optional, if specified spawns particle with 
+    --  specified material type param and disable z-buffer.
+    --  See also "minetest.ebf","minetest.emfn" and "minetest.eas" in "Helper functions".
+	vertical_frame_num = 16,
+    --  ^ optional, specify amount of vertical frames in texture.
+	first_frame = 0,
+    --  ^ optional, specify first frame to start animation.
+	frame_length = 0.1,
+    --  ^ optional, specify length of a frame in seconds. 
+    --  Set any value lower or equalt to 0 to disable animation.
+	loop_animation = true,
+    --  ^ optional, specify if animation should start from beginning after last frame.
+	glow = "#AAAF"
+    --  ^ optional, specify particle opacity and self-luminescence in darkness.
```

Comment:
@sfan5 ColorString or indexed color. Both are acceptable by parsing function. Notes added in Lua API doc.